src/aiq/front_ends/console/console_front_end_plugin.py

```diff
@@ -70,18 +70,18 @@ async def run(self):
 
             if logger.isEnabledFor(logging.INFO):
                 stream = StringIO()
-
                 self.full_config.print_summary(stream=stream)
 
                 click.echo(stream.getvalue())
 
-                workflow = builder.build()
-                session_manager = AIQSessionManager(workflow)
+            workflow = builder.build()
+            session_manager = AIQSessionManager(workflow)
 
             await self.run_workflow(session_manager)
 
     async def run_workflow(self, session_manager: AIQSessionManager = None):
```

Comment:
What if we also do not default to `None`?

```suggestion
    async def run_workflow(self, session_manager: AIQSessionManager):
```